subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -494,6 +495,17 @@ Since Java 11 no longer contains the `java.xml.bind` module, Gradle now bundles
 
 Please remove the `--add-modules java.xml.bind` option from `org.gradle.jvmargs`, if set.
 
+[[rel5.0:gradle_plugin_portal_metadata]]
+=== [5.0] `gradlePluginPortal()` repository now fetches artifact metadata from Maven POMs only
+
+The `gradlePluginPortal()` repository is now stricter when resolving artifact metadata, it doesn't infer the metadata from the sole presence of artifact files such as JARs. Only Maven POMs are now considered.
+
+This repository is used by default when resolving plugins from the `plugins {}` block.
+
+With this new behavior, if a plugin or a transitive dependency of a plugin found in the `gradlePluginPortal()` repository has no Maven POM it will fail to resolve.
+
+Artifacts published to a Maven repository without a POM should be fixed. If you encounter such artifacts, please ask the plugin or library author to publish a new version with proper metadata.
```

Comment:
There actually is a workaround by adjusting the repository in the pluginManagement block. May be worth pointing out for people who are stuck with a bad plugin.